src/Uno.UI.Runtime.Skia.Gtk/GtkCoreWindowExtension.cs

```diff
@@ -84,6 +87,62 @@ public GtkCoreWindowExtension(object owner)
 
 		partial void InitializeKeyboard();
 
+		internal static Fixed? FindNativeOverlayLayer(Gtk.Window window)
+		{
+			var overlay = (Overlay)((EventBox)window.Child).Child;
+			return overlay.Children.OfType<Fixed>().FirstOrDefault();
+		}
+
+		internal static Fixed? GetOverlayLayer(XamlRoot xamlRoot) =>
+			XamlRootMap.GetHostForRoot(xamlRoot)?.NativeOverlayLayer;
+
+		public bool IsNativeElement(object content)
+			=> content is Widget;
+
+		public void AttachNativeElement(object owner, object content)
```

Comment:
Is those new methods required to be public?

Reply:
It's an internal type, so that's fine.